specs/sonarqube/sonarqube.spec

```diff
@@ -0,0 +1,182 @@
+%define _loc_prefix       %{_prefix}/local
+%define _loc_exec_prefix  %{_loc_prefix}
+%define _loc_bindir       %{_loc_exec_prefix}/bin
+%define _loc_libdir       %{_loc_exec_prefix}/%{_lib}
+%define _loc_libdir32     %{_loc_exec_prefix}/%{_lib32}
+%define _loc_libdir64     %{_loc_exec_prefix}/%{_lib64}
+%define _loc_libexecdir   %{_loc_exec_prefix}/libexec
+%define _loc_sbindir      %{_loc_exec_prefix}/sbin
+%define _loc_bindir       %{_loc_exec_prefix}/bin
+%define _loc_datarootdir  %{_loc_prefix}/share
+%define _loc_includedir   %{_loc_prefix}/include
+%define _loc_mandir       %{_loc_datarootdir}/man
+%define _rpmstatedir      %{_sharedstatedir}/rpm-state
+%define _pkgconfigdir     %{_libdir}/pkgconfig
+
+%define __ldconfig        %{_sbin}/ldconfig
+%define __service         %{_sbin}/service
+%define __touch           %{_bin}/touch
+%define __chkconfig       %{_sbin}/chkconfig
+%define __updalt          %{_sbindir}/update-alternatives
+%define __useradd         %{_sbindir}/useradd
+%define __groupadd        %{_sbindir}/groupadd
+%define __getent          %{_bindir}/getent
+%define __sysctl          %{_sbindir}/sysctl
+%define __systemctl       %{_bindir}/systemctl
+
+################################################################################
+
+%define sonar_name       sonar-application
+%define short_name       sonar
+%define sonar_prefix     %{_opt}/%{name}-%{version}
+
+%define service_user     sonarqube
+%define service_group    sonarqube
+
+################################################################################
+
+Summary:              SonarQube Continuous Inspection
+Name:                 sonarqube
+Version:              7.9.1
+Release:              0%{?dist}
+License:              LGPLv3
+Group:                System Environment/Daemons
+URL:                  https://www.sonarqube.org/
+
+Source0:              https://github.com/SonarSource/%{name}/archive/%{version}.tar.gz
+Source1:              %{name}.service
+Source2:              %{name}.properties
+Source3:              %{name}.sysctl
+
+BuildRoot:            %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+
+BuildRequires:        java-11-openjdk java-11-openjdk-devel
+
+Requires:             java-11-openjdk java-11-openjdk-headless
+
+Requires(post):       systemd
+Requires(preun):      systemd
+Requires(postun):     systemd
+
+Provides:             %{name} = %{version}-%{release}
+
+ExclusiveArch:        x86_64
+
+################################################################################
+
+%description
+SonarQube provides the capability to not only show health of an application but
+also to highlight issues newly introduced. With a Quality Gate in place, you
+can fix the leak and therefore improve code quality systematically.
+
+################################################################################
+
+%prep
+%setup -qn %{name}-%{version}
+
+%build
+./gradlew build -DbuildNumber=1 -x test
```

Comment:
I think is better to use gradle from package, instead of wrapper script.

Reply:
Agree. I did not realize that `gradle` is already in this repository. Fixed (16dceb4).